src/cc_catalog_airflow/dags/common/storage/test_audio.py

```diff
@@ -0,0 +1,489 @@
+import logging
+import requests
+import pytest
+import tldextract
+
+from common.licenses import licenses
+from common.storage import audio
+
+logging.basicConfig(
+    format='%(asctime)s - %(name)s - %(levelname)s:  %(message)s',
+    level=logging.DEBUG)
+
+logger = logging.getLogger(__name__)
+
+# This avoids needing the internet for testing.
+licenses.urls.tldextract.extract = tldextract.TLDExtract(
+    suffix_list_urls=None
+)
+audio.columns.urls.tldextract.extract = tldextract.TLDExtract(
+    suffix_list_urls=None
+)
+
+mock_audio_args = {
+    'foreign_landing_url': 'https://landing_page.com',
+    'audio_url': 'http://audiourl.com',
+    'license_': 'by-nc',
+    'license_version': '1.0',
+    'license_url': None,
+    'foreign_identifier': 'foreign_id',
+    'thumbnail_url': 'http://thumbnail.com',
+    'duration': 200,
+    'creator': 'tyler',
+    'creator_url': 'https://creatorurl.com',
+    'title': 'agreatpicture',
+    'meta_data': None,
+    'raw_tags': None,
+    'bit_rate': None,
+    'sample_rate': None,
+    'category': None,
+    'genre': None,
+    'audio_set': None,
+    'alt_audio_files': None,
+    'source': 'testing_source',
+    'ingestion_type': 'provider_api',
+
+}
```

Comment:
Do you think it would be better to mock fields that are list/arrays as empty ones instead of `None`? I don't know `/shrug` so thought I'd ask anyway

Reply:
This will make the tests more robust, thank you!